config/_default/menus/menus.en.yaml

```diff
@@ -1013,6 +1013,11 @@ main:
     parent: infrastructure_livecontainers
     identifier: infrastructure_livecontainers_configuration
     weight: 301
+  - name: Container Images
```

Comment:
Should we call it "Container Images View" to keep it consistent with "Containers View" and follow the same pattern of the blog post?

https://www.datadoghq.com/blog/datadog-container-image-view/

Reply:
Yes great idea!